tools/generator/src/Generator+CreateProducts.swift

```diff
@@ -25,11 +25,13 @@ extension Generator {
                     fileType = target.product.type.fileType
                 }
 
+                let bestTarget = target.sortedTargets.first!
```

Comment:
How is this target the best target? Depending on your answer, either renaming this variable or adding a comment could help.

Reply:
There is a [comment here](https://github.com/buildbuddy-io/rules_xcodeproj/blob/main/tools/generator/src/Generator%2BConsolidateTargets.swift#L242-L245) that describes the "best" concept.

Reply:
:+1:. I also renamed it to `defaultTarget` at this location and added another comment, just for extra clarity.